src/utils/preprocessing.py

```diff
@@ -406,3 +406,13 @@ def standing_laying_data_preprocessing(source_path):
     img = tf.image.resize(img, [240, 180])
     img = tf.expand_dims(img, axis=0)
     return img
+
+
+def sample_systematic_from_artifacts(artifacts: list, n_artifacts: int) -> list:
```

Comment:
Please reference the original code in cgm-ml such that we can adjust both sides when one side changes (e.g. in docstring)